chapter-1-basics.md

```diff
@@ -360,10 +385,16 @@ And finally, the method that gets called on a cell after completing work is
 public override void SequenceCompleted(SequenceCompleted completed)
 {
     _currentSession = completed;
-
-    var rtw = Session.StartSession(ActivityClassification.Production, ReadyToWorkType.Push);
-    PublishReadyToWork(rtw);
-    _currentSession = rtw;
+    if (Driver == null)
```

Comment:
Again: Template?

I am not sure if we need the driver and/or visual instructor at this point.